services/user.py

```diff
@@ -0,0 +1,46 @@
+from db.models import User

```

Comment:
According to the checklist, it's better to use `get_user_model()` instead of `User`. It's a best practice in Django, because it allows us to swap out the user model with a custom one.